examples/basic/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hyphen-ai/openfeature-provider-go/pkg/toggle"
+	"github.com/open-feature/go-sdk/openfeature"
+)
+
+func main() {
+	// Configuration for the Hyphen provider
+	config := toggle.Config{
+		Application:       "application-id",
+		Environment:       "production",
+		PublicKey:         "your-public-key",
+		HorizonServerURLs: []string{"https://horizon.hyphen.ai"},
```

Comment:
Let's not include this in the example since most people will not need to set it.